functions/zram.bash

```diff
@@ -4,13 +4,11 @@ init_zram_mounts() {
     local TAG=openhabian_v1.5
     TMP="$(mktemp -d /tmp/.XXXXXXXXXX)"
 
-    /usr/bin/git clone --branch "$TAG" "$ZRAMGIT" "$TMP"
-    cd ${TMP}
-    /usr/bin/git fetch --tags
-    /usr/bin/git checkout ${TAG}
+    /usr/bin/git -q clone --branch "$TAG" "$ZRAMGIT" "$TMP"
+    cd "${TMP}" || exit
```

Comment:
Why `cd`?

Reply:
because the following command needs to be executed in there.
I moved this line down by one so it's in the block it belongs to now